app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 class User < ApplicationRecord
   attr_accessor :remember_token
   before_save {email.downcase!}
+  paginates_per Settings.models.user.pag
```

Comment:
như này là toàn bộ model, lỡ 1 trang anh thích là 10 bản ghi, 1 trang anh  thích 5 bản ghi thì em  làm như thế nào?

Reply:
Em sửa lại rồi đấy ạ